src/modules/Search/Search.js

```diff
@@ -57,7 +57,10 @@ export default class Search extends Component {
     noResultsDescription: PropTypes.string,
 
     /** Message to display when there are no results. */
-    noResultsMessage: PropTypes.string,
+    noResultsMessage: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.object,
+    ]),
```

Comment:
I believe this should just be `PropTypes.node` only.  `PropTypes.node` includes any renderable value: string, number, element, or an array (or fragment) of any of these.  However, objects are not renderable.

Let's also update the typings in `Search.d.ts` to reflect the type change as well.